shells/chrome/src/panel.js

```diff
@@ -17,12 +17,36 @@ var inject = require('./inject');
 
 import type {Props} from '../../../frontend/Panel';
 
-// chrome.devtools.panels added in Chrome 18.
-// chrome.devtools.panels.themeName added in Chrome 54.
-const browserName = 'Chrome';
-const themeName = (chrome.devtools.panels : any).themeName === 'dark'
-  ? 'ChromeDark'
-  : 'ChromeDefault';
+const IS_CHROME = navigator.userAgent.indexOf('Firefox') < 0;
```

Comment:
Is this too clunky? Should I try to inject this parameter via the build script (or similar)?